Procfile.dev

```diff
@@ -0,0 +1,2 @@
+web: bin/rails server -p 3000
```

Comment:
The Procfile isn't actually used if you wanted to steal that.

I doubt we'll ever use it for live envs

Reply:
I left that as this is purely a dev change, but another PR should remove it if it's not used.